### ISSUE TITLE
Not syncing with other computers

### ISSUE THREAD

**Issue by ftodd1** [sienori/Tab-Session-Manager#852]
Currently I am using TSM (latest version) on 2 home computers with Firefox.  I have them both set up to sync with Firefox just fine, both linked to the same account.  However, they are both set up with different computer users name in TSM one has Tv-Room title and other Maincomp.

They both sync up fine with the net work, and sessions are saved and backed up routinely.   However, in session backups with TSM history, it only shows the session(s) for each computer and not both of them.

**Issue:**  
On the Tv-Room there are no user saved, or auto-synced sessions for Maincomp listed and vs versa.  Firefox syncs tabs, bookmarks, recent history with no problem.  I can retrieve  the data which was stored for both computers with this method.
.      
OS System set-up: 
Windows 7 Ultimate, running on both computers (still need to run a specialized program)
Browsers used:
Tv-Room computer runs Firefox beta 92.x, 
Maincomp Firefox Quantum 91.x (the latest stable version of FF)

Any thoughts or help would be appreciated.

Thanks for your time and making awesome add-on!

**Comment by sienori** [comment]
Have you enabled the Cloud Sync option?
Please sign in to Google from the TSM settings page.
![image](https://user-images.githubusercontent.com/29514424/132484807-50173578-9530-45ff-b538-22b7cf38771b.png)

**Comment by ftodd1** [comment]
Yes I have, but the only one that shows up is each individually.  So, ManiComp shows on MainComp compter but not TvRoom and TvRoom show up on the TvRoom computer but not MainComp.  Firefox user is same on both pages FF sync history and bookmarks are linked just fine with both showing up.  So If I look and bookmark github on MainComp it will be listed on the TvRoom computer history.  But the sessions are not like that.
It's a puzzle??? :-)
